app/celery/tasks.py

```diff
@@ -1,21 +1,19 @@
 from flask import current_app
 
 from app import notify_celery
-from app.render import alerts_from_yaml, get_rendered_pages
-from lib.utils import purge_cache, upload_to_s3
+from app.models.alerts import Alerts
+from app.render import get_rendered_pages
+from app.utils import purge_cache, upload_to_s3
 
 
 @notify_celery.task(bind=True, name="publish-govuk-alerts", max_retries=20, retry_backoff=True, retry_backoff_max=300)
 def publish_govuk_alerts(self):
-
     try:
-        alerts = alerts_from_yaml()
+        alerts = Alerts.load()
         rendered_pages = get_rendered_pages(alerts)
 
-        upload_to_s3(current_app.config, rendered_pages)
-
-        purge_cache(current_app.config)
-    except Exception:
-        current_app.logger.exception("Failed to publish content to gov.uk/alerts: " + Exception)
+        upload_to_s3(rendered_pages)
+        purge_cache()
+    except Exception as e:
+        current_app.logger.exception("Failed to publish content to gov.uk/alerts: " + str(e))
```

Comment:
Could we get away with just `current_app.logger.exception("Failed to publish content to gov.uk/alerts")` here? `logger.exception` should give the stack trace so I'm not sure if adding `str(e)` to the message gives anything extra.

Reply:
Ah yes, and the stacktrace includes the `str()` of the exception 👍.

Updated in https://github.com/alphagov/notifications-govuk-alerts/pull/163/commits/f1173fbf5ee38157819cb0282996265bccde9a29.